src/pytest_dbfixtures/factories/dynamodb.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (C) 2013 by Clearcode <http://clearcode.cc>
+# and associates (see AUTHORS).
+
+# This file is part of pytest-dbfixtures.
+
+# pytest-dbfixtures is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# pytest-dbfixtures is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public License
+# along with pytest-dbfixtures.  If not, see <http://www.gnu.org/licenses/>.
+import pytest
+
+from pytest_dbfixtures.executors import TCPExecutor
+from pytest_dbfixtures.utils import try_import, get_process_fixture
+
+
+def dynamodb_proc(jar_path=None, host=None, port=None):
```

Comment:
> Why are these `None` by default? shouldn't `jar_path` always be provided?
User has to download a package with DynamoDB and we do not know where it is

> or at least it should check some default places. `host` and `port` could also have some sane values
Yeah, it can be host='hostname' and port=8000